wikichange/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "wikichange",
+    "description": "Annontate changes on Wikipedia pages",
+    "version": "0.0.1",
+    "homepage": "http://chrome.google.com/webstore",
+    "author": "ENTER YOUR NAME HERE",
+    "repository": {
+        "type": "git",
+        "url": "ENTER GIT REPO URL"
+    },
+    "scripts": {
+        "start": "xt-build -e dev -w",
+        "start:firefox": "xt-build -e dev -p firefox -w",
+        "build": "xt-build -e prod",
+        "build:firefox": "xt-build -e prod -p firefox",
+        "clean": "xt-clean",
+        "docs": "xt-docs",
+        "test": "xt-test",
+        "coverage": "nyc --reporter=lcov npm run test",
+        "sync": "xt-sync"
+    },
+    "babel": {
+        "presets": [
+            "@babel/preset-env"
+        ]
+    },
+    "eslintIgnore": [
+        "test/**/*"
+    ],
+    "devDependencies": {
+        "extension-cli": "latest"
+    },
+    "xtdocs": {
+        "source": {
+            "include": [
+                "README.md",
+                "src"
+            ]
+        }
+    },
+    "xtbuild": {
+        "js_bundles": [
+            {
+                "name": "background",
+                "src": "./src/background.js"
+            },
+            {
+                "name": "content",
+                "src": "./src/content/*.js"
+            },
+            {
+                "name": "popup",
+                "src": "./src/popup/popup.js"
+            }
+        ]
+    },
+    "dependencies": {
+        "chart.js": "^3.9.1"
```

Comment:
we don't need dependencies for fetch anymore because there is a fetch client side right?

Reply:
Yes